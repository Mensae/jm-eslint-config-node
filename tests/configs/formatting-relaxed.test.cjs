/* eslint-env jest */

const formattingConfig = require('../../configs/formatting-relaxed.cjs');

describe('formatting-relaxed', () => {
  it('should match the snapshot', () => {
    expect(formattingConfig).toMatchSnapshot();
  });
});
