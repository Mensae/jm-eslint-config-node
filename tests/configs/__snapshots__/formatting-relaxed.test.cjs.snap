// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`formatting-relaxed should match the snapshot 1`] = `
{
  "env": {
    "es2022": true,
    "node": true,
  },
  "parserOptions": {
    "ecmaVersion": "latest",
    "sourceType": "module",
  },
  "plugins": [
    "n",
  ],
  "reportUnusedDisableDirectives": true,
  "rules": {
    "n/callback-return": "off",
    "n/exports-style": "warn",
    "n/file-extension-in-import": "warn",
    "n/global-require": "warn",
    "n/no-mixed-requires": "warn",
    "n/no-process-env": "off",
    "n/no-restricted-import": "off",
    "n/no-restricted-require": "off",
    "n/no-sync": "off",
    "n/prefer-global/buffer": "warn",
    "n/prefer-global/console": "warn",
    "n/prefer-global/process": "warn",
    "n/prefer-global/text-decoder": "warn",
    "n/prefer-global/text-encoder": "warn",
    "n/prefer-global/url": "warn",
    "n/prefer-global/url-search-params": "warn",
    "n/prefer-promises/dns": "warn",
    "n/prefer-promises/fs": "warn",
  },
}
`;
